src/features/chat/ui/ChatDialogOpenButton.tsx

```
@@ -0,0 +1,9 @@
+'use client';
+
+import useChatDialogTrigger from '../hooks/useChatDialogTrigger';
+
+export default function ChatDialogOpenButton() {
+  const { openChatDialog } = useChatDialogTrigger();
+
+  return <button onClick={openChatDialog}>이거 채팅버튼임</button>;
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)